types/tusk-mastodon/index.d.ts

```diff
@@ -0,0 +1,32 @@
+declare class TuskMastodonInstance {
+    constructor(options: TuskMastodonOptions);
+
+    put(path: string, params: GeneralObject): Promise<GeneralObject>;
+    get(path: string, params: GeneralObject): Promise<GeneralObject>;
+    post(path: string, params: GeneralObject): Promise<GeneralObject>;
+    patch(path: string, params: GeneralObject): Promise<GeneralObject>;
+    delete(path: string, params: GeneralObject): Promise<GeneralObject>;
+
+    request(method: ValidHTTPMethod, path: string, params: GeneralObject): Promise<GeneralObject>;
+
+    getAuth(): { "access_token": string };
+    setAuth(auth: { "access_token": string }): void;
+
+    formEncodeParams(params: GeneralObject): string;
+}
+
+interface TuskMastodonOptions {
+    "access_token": string;
+    /**
+     * The base URL of the Mastodon instance to connect to. Defaults to `https://mastodon.social/api/v1/`.
+     */
+    "api_url": string;
```

Comment:
this one looks like optional:
https://github.com/skullzarmy/Tusk/blob/master/lib/mastodon.js#L19
```suggestion
    "api_url?": string;
```

Reply:
Fixed in 3c1885d5cfeaf2f22b1e560da62793bf14d3cea4